src/settings/NoticePolicy/utils/get-templates.js

```diff
@@ -1,8 +1,8 @@
 import { reduce } from 'lodash';
 
 export default (templates = [], categoryName = '') => {
-  return reduce(templates, (items, { id, name, category }) => {
-    if (category === categoryName) {
+  return reduce(templates, (items, { id, name, category, active }) => {
+    if (category === categoryName && active) {
```

Comment:
I would suggest checking for active ones first. Additionally, it will look a bit neater What do you think?
```suggestion
    if (active && category === categoryName) {
```

Reply:
Funny, at first I wrote like you're suggesting and then changed it conversely. But I agree with you.